test/generate_new_gameboard_test.rb

```diff
@@ -26,9 +29,86 @@ def test_initialize_empty_gameboard_spaces_return_values
   def test_generate_blank_gameboard_return_values
     new_gameboard = GenerateNewGameboard.new(4)
     blank_gameboard = new_gameboard.generate_blank_gameboard
+
     assert_equal Array, blank_gameboard.class
     assert_equal 4, blank_gameboard.length
     assert_equal Hash, blank_gameboard.first.class
     assert_equal 4, blank_gameboard.first.length
+    assert_equal ["a1", "a2", "a3", "a4"], blank_gameboard[0].keys
```

Comment:
Note to self - don't mess around with spacing so much between commits. It gets hard to see what you've actually worked on if there's a problem. If you're not actively working on it: don't touch it.